### PR TITLE
CDPT-1246 AS3CF increase batch size and add logging

### PR DESCRIPTION
**Pull request by EarthlingDavey**
This PR 

- adds filters to increase batch size for the AS3CF plugin.
- Adds a post instal/update script to that patches in logging to the plugin - similar to Justice UK patching technique.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -113,18 +113,20 @@ RUN make test
...
+
 ARG COMPOSER_USER
 ARG COMPOSER_PASS
 ARG AS3CF_PRO_USER
 ARG AS3CF_PRO_PASS
 
 COPY --from=composer:2 /usr/bin/composer /usr/bin/composer
 
-WORKDIR /var/www/html
+COPY ./bin/composer-auth.sh ./bin/composer-post-install.sh ./bin/
```
Comment (wilson1000) [inline Dockerfile:126]: Can you pop `WORKDIR /var/www/html` on line 132 so `exec` can land us in the right place?

Comment (EarthlingDavey) [reply Dockerfile:126]: Are you ok with it being a few lines up to match what we have on Justice UK? It's on 11 at the mo.

Comment (wilson1000) [reply Dockerfile:126]: Mate, wherever you feel is best.

Comment (wilson1000) [reply Dockerfile:126]: I didn't see it there lol

**Review by wilson1000 — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (EarthlingDavey) [reply Dockerfile:126]: haha, I was trying to simplify things, just made it more complicated 😅

**Pull request merged**